src/main/scala/com/databricks/spark/xml/util/XSDToSchema.scala

```diff
@@ -25,18 +25,19 @@ import org.apache.ws.commons.schema._
 import org.apache.ws.commons.schema.constants.Constants
 
 /**
- * Utility to generate a Spark schema from an XSD. Not all XSD schemas are simple tabular schemas,
- * so not all elements or XSDs are supported.
- */
+  * Utility to generate a Spark schema from an XSD. Not all XSD schemas are simple tabular schemas,
```

Comment:
Revert these if you would - might be easiest to change your indent settings in the IDE